owncloudDomain/src/main/java/com/owncloud/android/domain/exceptions/ResourceLockedException.kt

```diff
@@ -0,0 +1,5 @@
+package com.owncloud.android.domain.exceptions
+
+import java.lang.Exception
+
+class ResourceLockedException : Exception()
```

Comment:
Here should be a line break in the end of the class. And also add the license text with your name.

Reply:
Indeed there is a line break, otherwise GitHub would indicate it

Reply:
I did have the line break not sure why it isn't showing here. I added the license header.